### ISSUE TITLE
Clicking "I agree" button don't hide the cookie-window & how to change text in message?

### ISSUE THREAD

**Issue by allinws** [TyMaszWeb/django-cookie-law#77]
When I click "I agree" nothing happens, but if I reload the window the warning is gone. Don't work on either mobile or desktop  and I tried with various browsers, still same problem - any idea what the issue can be? I don't have any warnings in the browser console.

Also, I'm wondering how to change the text in the warning to my own language? I did a PR with swedish translation but haven't seen any action on it yet.

Thanks.

Versions
django-cookie-law==2.1.1
Django==3.2.9

In my base.html

``` 
{% load cookielaw_tags %}


{% if cookielaw.accepted %}
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-KZLXQRFD87"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-KZLXQRFD87');
    </script>
  {% endif %}

  <script src="{% static 'cookielaw/js/cookielaw.js'%} "></script>


```

in my settings

```
added in installed apps

        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
                'django_admin_env_notice.context_processors.from_settings',
                'cookielaw.context_processors.cookielaw'
            ],
```

**Comment by piotrkilczuk** [comment]
Is this with or without jquery? @allinws

**Comment by minifisk** [comment]
> > I'm wondering how to change the text in the warning to my own language?
> 
> It should be possible to do this even if your MR is not merged.
> 
> The answer should be here: https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#localization-how-to-create-language-files

Thanks very much for your quick reply. The issue was that I was using a limited version of Jquery that didn't have the "slidedown" method. Now it works, thanks again :)

**Comment by piotrkilczuk** [comment]
Oh, nice!

Can you let me know precisely what version? Maybe I can extend the test suite to use that version too...

**Comment by piotrkilczuk** [comment]
https://github.com/TyMaszWeb/django-cookie-law/pull/78 should be fixing this

**Comment by piotrkilczuk** [comment]
Released as 2.2.0. Any chance you can verify this @minifisk ?
